providers/openfeature-provider-flagd/README.md

```diff
@@ -5,6 +5,7 @@ This provider is designed to use flagd's [evaluation protocol](https://github.co
 ## Installation
 
 ```
+pip install openfeature-sdk
```

Comment:
We are going to assume in README files that users have already configured the SDK or know how to do so. This is not the best place for a tutorial on OpenFeature.

See the examples in the Java or JS SDK https://github.com/open-feature/java-sdk-contrib/tree/main/providers/flagd

Reply:
Hmm, I disagree. As an occasional user, I got here from a Google search and it didn't "just work". So off I go to Google and putting the pieces together myself, I eventually got it working. But it wasn't a good / fun UX.